server/src/main/java/org/spine3/server/reflect/EventHandlerMethod.java

```diff
@@ -117,25 +111,24 @@ public EventHandlerMethod create(Method method) {
 
         private enum Singleton {
             INSTANCE;
-            @SuppressWarnings("NonSerializableFieldInSerializableClass")
-            private final Factory value = new Factory();
+            @SuppressWarnings({"NonSerializableFieldInSerializableClass", "UnnecessarilyQualifiedInnerClassAccess"})
```

Comment:
Please demote this warning to Info level and remove suppression here and in other places.